### ISSUE TITLE
bitwarden/mssql restart loop

### ISSUE THREAD

**Issue by derjoerg** [bitwarden/server#778]
Hi,

I just realised, that I'm not able anymore to login to my self-hosted bitwarden installation. Browser-Plugin and Android-App still working as expected, but when I try to login through the web-browser I get the error-message: "An unexpted error occured".

"docker container ls" revealed, that the mssql-container is restarting, and "docker logs bitwarden-mssql" shows the following:
/opt/mssql/bin/sqlservr: The file archive [/opt/mssql/lib/system.netfx.sfp] is invalid.  Please resolve and try again.

bitwarden/mssql is running on 1.34.0

**Comment by kspearrin** [comment]
What host OS are you using?

**Comment by derjoerg** [comment]
Pretty old Ubuntu 14.04.6

**Comment by martinresch** [comment]
Hi, same here on Ubuntu 18.04.3 LTS.

**Comment by vncloudsco** [comment]
Hi, same here on Ubuntu 18.04.3 LTS bitwarden/mssql not working
bitwarden/mssql:1.34.0 auto restart when we execute the query. 

```
root@bitwarden:~# cat /etc/os-release
NAME="Ubuntu"
VERSION="18.04.4 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.4 LTS"
VERSION_ID="18.04"
```

**Comment by cmmh** [comment]
Having the same problem on Ubuntu 20.04.1

root@neo:~$ cat /etc/os-release 
NAME="Ubuntu"
VERSION="20.04.1 LTS (Focal Fossa)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 20.04.1 LTS"
VERSION_ID="20.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=focal
UBUNTU_CODENAME=focal

e328ba183635        bitwarden/mssql:1.36.1           "/entrypoint.sh"    15 minutes ago      Restarting (1) 31 seconds ago                                                           bitwarden-mssql

Loos like this issue has been open for 2 months.  Any resolution on the horizon?

**Comment by aschlick** [comment]
I started getting this issue too after updating:
```
$ docker logs bitwarden-mssql
/opt/mssql/bin/sqlservr: The file archive [/opt/mssql/lib/system.netfx.sfp] is invalid.  Please resolve and try again.
/opt/mssql/bin/sqlservr: The file archive [/opt/mssql/lib/system.netfx.sfp] is invalid.  Please resolve and try again.
/opt/mssql/bin/sqlservr: The file archive [/opt/mssql/lib/system.netfx.sfp] is invalid.  Please resolve and try again.
/opt/mssql/bin/sqlservr: The file archive [/opt/mssql/lib/system.netfx.sfp] is invalid.  Please resolve and try again.
/opt/mssql/bin/sqlservr: The file archive [/opt/mssql/lib/system.netfx.sfp] is invalid.  Please resolve and try again.
```

I've tried rebuilding, stopping and `docker prune` and starting again, and trying to run bash on the mssql image, but none of these things seem to help or get me to a place to fix it.

**Comment by Mart124** [comment]
Server `1.38.2` might solve some mssql restart loop issues, thanks to fix #1030 (solving #1020).

**Comment by djsmith85** [comment]
@derjoerg, @martinresch, @vncloudsco, @cmmh, @aschlick : I know this is pretty old but did an update to a newer version as mentioned by @Mart124 resolve the issues?
